src/ai_marketplace_monitor/webui/found_export.py

```
@@ -0,0 +1,120 @@
+                    details_by_key[(value["marketplace"], value["id"])] = value
+            elif tag == CacheType.AI_INQUIRY.value and len(key) >= 4:
+                value = local_cache.get(key)
+                if isinstance(value, dict):
+                    rating_by_hash[key[3]] = value
+            elif tag == CacheType.USER_NOTIFIED.value and len(key) >= 4:
+                date, listing_hash, price = _normalize_notified(local_cache.get(key))
+                notified.append((key[1], key[2], key[3], date, listing_hash, price))
+        except KeyboardInterrupt:
+            raise
+        except Exception:
+            continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)